content/en/guides/internals-glossary/context.md

```diff
@@ -113,6 +113,14 @@ A valid workaround would be using `window.onNuxtReady(() => { window.$nuxt.$rout
 
 </base-alert>
 
+<base-alert type="info">
+
+A client-side call to `redirect` always throws the error `ERR_REDIRECT`. This is to block Nuxt from doing any tasks between the time redirect call happens until the browser starts navigation.
```

Comment:
Actually, if you look at [this block](https://github.com/nuxt/nuxt.js/blob/c15078fa648ac5a4c77241700e9f1bd74a802af8/packages/vue-app/template/utils.js#L213-L219), the `ERR_REDIRECT` is only called on client-side **when redirecting to an external URL**.